README.md

```diff
@@ -58,7 +58,7 @@ resumeSchema.validate({ name: "Thomas" }, function (err, report) {
 
 More likely
 
-```
+```js
 var fs = require('fs');
 var resumeSchema  = require('resume-schema');
 var resumeObject = JSON.parse(fs.readFileSync('resume.json', 'utf8'));
```

Comment:
Not sure, is or not, if it/he/she keep doing this(https://github.com/jsonresume/resume-schema/issues/213), I'll report the abuse behavior.